
hrl_manipulation_task:

  # Main manipulator
  arm: l

  # For auditory perception
  head_audio_frame: head_mount_kinect_ir_link

  # bowl center                                                                                                    
  sub_ee_frame: r_gripper_tool_frame
  sub_ee_pos_offset:
    x: 0.11
    y: 0
    z: 0.04
  sub_ee_orient_offset:
    rx: 0
    ry: 0
    rz: 1.570795

  # For visual perception
  scooping:
    artag_id: 
       - 9
    artag_length: 0.084
    artag_max_id: 18
    artag_total_tags: 1
    artag_buf_size: 10
  
  # Mouth pose limits (wrt torso lift link), xyz order
  mouth_pos_limits:
    max: [1.2, 1.0, 0.4]
    min: [0.5, 0.0, -0.4]
